### PR TITLE
bug fixes for TabBar refactor

### PR DESCRIPTION
**Pull request by Inbal-Tish**
## Description
Fix refactor bugs.

## Changelog
TabBar - fix refactor bugs

### CODE REVIEW

File: src/components/scrollBar/index.tsx [hunk 1]
```diff
@@ -25,7 +25,7 @@ export interface ScrollBarProps extends FlatListProps<any> {
     /**
      * The element to use as a container, instead of a View
      */
-    containerView?: JSX.Element,
+    containerView?: object,
```
Comment (ethanshar) [inline src/components/scrollBar/index.tsx:28]: Is that right? 
Shouldn't `containerView` be an element?

**Review by ethanshar — COMMENTED** [review]
I noticed you don't push changes in generateTypes. 
Make sure to always run `npm run build:dev`. It verifies there are no TS errors and also create the updated generatedTypes

File: src/components/scrollBar/index.tsx (hunk 1 above)
Comment (Inbal-Tish) [reply src/components/scrollBar/index.tsx:28]: No. Since we're not passing <TabletView/>, we're passing TabletView.

**Comment by Inbal-Tish** [comment]
> I noticed you don't push changes in generateTypes.
> Make sure to always run `npm run build:dev`. It verifies there are no TS errors and also create the updated generatedTypes

Done

File: src/components/scrollBar/index.tsx (hunk 1 above)
Comment (ethanshar) [reply src/components/scrollBar/index.tsx:28]: Still it's not a object. 
I think there's a dedicate type for it.. 

Try searching for the right prop.

Comment (Inbal-Tish) [reply src/components/scrollBar/index.tsx:28]: I couldn't find it, the 'object' type is the only one that doesn't error...

**Review by ethanshar — APPROVED** [review]

**Pull request merged**